filebeat/crawler/prospector_log.go

```diff
@@ -140,6 +140,9 @@ func (p *ProspectorLog) scanGlob(glob string) {
 		// Track the stat data for this file for later comparison to check for
 		// rotation/etc
 		p.prospectorList[h.Path] = *h.Stat
+		// TODO: Registrar now updated also if no changes happened. Improve efficiency here
+		// Update registrar
+		p.Prospector.registrar.Persist <- h.GetState()
```

Comment:
BTW: I'm not happy at all how the state works at the moment, but I will do a heavier refactoring in master later, but want to keep this "backportable".